protocol/src/main/protobuf/UserBitShared.proto

```diff
@@ -298,3 +298,17 @@ enum CoreOperatorType {
   NESTED_LOOP_JOIN = 35;
   AVRO_SUB_SCAN = 36;
 }
+
+message Func {
+  optional string name = 1;
+  repeated common.MajorType major_type = 2;
+}
+
+message Jar {
```

Comment:
A Jar contains one or more functions?
The name is the file name? Which, presumably, the user must ensure is unique across statically defined UDFs (those in the class path) and dynamic UDFS?